docs/military/townsville.md

```diff
@@ -4,7 +4,78 @@
+| Townsville ADC    | Townsville Tower  | 118.300         | TL_TWR        |
+| Townsville SMC    | Townsville Ground  | 121.800         | TL_GND        |
+| Townsville ACD    | Townsville Delivery  | 128.100         | TL_DEL       |
+| Townsville ATIS    |   | 133.500         | YBTL_ATIS       |
+
+## Airspace
+### TCU
+TL TCU owns the Class C and G airspace within 36 DME TL from `SFC` to `F180`
+
+<figure markdown>
+![TL TCU/ADC Airspace Structure](img/tltcu.png){ width="2000" }
+  <figcaption>TL ADC/TCU Airspace Structure</figcaption>
+</figure>
+
+### ADC
+TL ADC owns the Class C arispace in the TL CTR (Extends to 7NM of the thresholds of 01 and 19.) from `SFC` to `A015`. This airsapce is designed to facilitate Helicopter Traffic and Ciruit Traffic.
+
+## Aerodrome
+### Runway Operations
+Runways 01 and 19 are the primary runways at Townsville. 
+Runway 07 is often used in conjunction with 01 for VFR arrivals and Helicopter Traffic. 
+
+### NON-RNAV Departures
+For non-RNAV approved IFR aircaft with a wake turbulence category of medium or greater, issue a RADAR SID.
+For non-RNAV approved IFR aircraft with a wake turbulence category of light, issue a visual departure or Radar SID
+
+## Charts
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Miscellaneous
+### Circuit Operations
+VFR aircraft conducting circuits inside TL ADCs airspace shall be at or below `A015`. Higher circuit altitude may be given if voice coordinated with TL TCU.
+
+### Circuit Direction
+| Runway | Direction |
+| ------ | ----------|
+| 01     | Left  |
+| 07     | Left |
+| 19     | Right |
+| 25     | Right |
+
+### Helicopter Procedures
+Procedures relating to Helicopters can be found in the Airforce AIP AD2 SUPPS Section for Townsville. (7.5 - 7.9)
+
+## Coordination
+### Enroute
+#### Departures
+The Standard assignable level from TL TCU to TBP is the lower of `F180` or the `RFL`. All other aircraft must be voice coordinated. 
+
+!!! example
+    <span class="hotline">**TL TCU** -> **TBP**</span>: "JST729, with your concurrence, will be assigned `F160`, for my separation with QLK327D"  
+    <span class="hotline">**TBP** -> **TL TCU**</span>: "JST729, concur `F160`"  
+
+#### Arrivals
+The Standard assignable level from KEN(TBP) to TL TCU is `A100`, and assigned a STAR. All other aircraft must be voice coordinated.
+
+### TL ADC
+#### Auto Release
+"Next" coordination is **not** required to TL TCU for aircraft that are:  
+a) Departing from a runway nominated on the ATIS; and  
+b) Assigned the standard assignable level (`F180`); and  
```

Comment:
Is this confirmed correct? I know that's how they do it at YPDN but is it the same at YBTL?

Also can the standard assignable level be presented in a format consistent with other pages? Eg:

The Standard Assignable level from SY ADC to SY TCU is:
For Jets: A050
For Non-Jets: The lower of A030 or the RFL

Reply:
@Augzer before we merge, did we get any clarification on the source of this (re: Alex's comment above)?